rca/static_src/sass/components/_tab-item.scss

```diff
@@ -58,4 +65,21 @@
             color: $color--black;
         }
     }
+
+    .js-is-sticky.jump-nav & {
+        color: $color--meta-lightbg;
+
+        &:hover {
+            color: $color--black;
+        }
+
+        &.is-active {
+            color: $color--black;
```

Comment:
meganitpick: could combine these up to save a few lines 

```
        &:hover,
        &.is-active {
            color: $color--black;
        }
```

Reply:
👍